addons/web/static/src/legacy/frontend/rainbowman/rainbow_man.js

```diff
@@ -25,6 +25,7 @@ var RainbowMan = Widget.extend({
      * @constructor
      * @param {Object} [options]
      * @param {string} [options.message] Message to be displayed on rainbowman card
+     * @param {string} [options.messageIsHtml=false] If true, message is considered as HTML otherwise it's escaped
```

Comment:
I'd say leave the doc off this will probably be stripped if / when `Markup` gets merged.

Reply:
I leave the doc.
At least we'll know that this option exists and why it is used for while `Markup` is not merged.

Reply:
I still think leaving it out is a feature: discouraging users from using this is a good thing.